packages/analytics/__tests__/analytics.test.ts

```diff
@@ -843,10 +877,35 @@ describe('Analytics', function () {
       expect(initiateOnDeviceConversionMeasurementWithEmailAddress).toBeDefined();
     });
 
+    it('`initiateOnDeviceConversionMeasurementWithHashedEmailAddress` function is properly exposed to end user', function () {
+      expect(initiateOnDeviceConversionMeasurementWithHashedEmailAddress).toBeDefined();
+    });
+
+    it('`initiateOnDeviceConversionMeasurementWithHashedEmailAddress` throws if not a string', function () {
+      expect(() =>
+        // @ts-ignore
+        firebase.analytics().initiateOnDeviceConversionMeasurementWithHashedEmailAddress(true),
```

Comment:
@matinzd - I mean modular API like this:
```suggestion
	initiateOnDeviceConversionMeasurementWithHashedEmailAddress(getAnalytics(), true),
```

Reply:
Yeah sorry :) Done.